File: csharp/DVSClient/DVSClient/server/address/search/RestApiSearchResultAdditionalAttribute.cs
using Newtonsoft.Json;

namespace DVSClient.Server.Address.Search
{
    public class RestApiSearchResultAdditionalAttribute
    {
        [JsonProperty("name")]
        public string? Name { get; set; }

        [JsonProperty("value")]
        public string? Value { get; set; }
    }
}

File: java/src/main/java/com/experian/dvs/client/server/address/search/RestApiSearchResultAdditionalAttribute.java
package com.experian.dvs.client.server.address.search;

import com.fasterxml.jackson.annotation.JsonProperty;

public class RestApiSearchResultAdditionalAttribute {

    @JsonProperty("name")
    private String name;

    @JsonProperty("value")
    private String value;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getValue() {
        return value;
    }

    public void setValue(String value) {
        this.value = value;
    }
}
